### PR TITLE
feat(form-label): Component Label and FormLabel #181

### PR DESCRIPTION
**Pull request by airikej**

### CODE REVIEW

**Review by mart-sessman — APPROVED** [review]

File: libs/react-components/tedi/src/components/form/form-label/form-label.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  renderWithoutLabel?: boolean;
+  /**
+   * Specifies the size of the label text.
+   * Options include 'small' for a smaller label size or 'default' for the standard size.
+   * @default default
+   */
+  size?: 'small' | 'default';
+}
+
+export const FormLabel = (props: FormLabelProps): JSX.Element => {
```
Comment (kasparsuvi1) [inline libs/react-components/tedi/src/components/form/form-label/form-label.tsx:44]: Is there some problems why we cant use this component inside community form components?

Comment (airikej) [reply libs/react-components/tedi/src/components/form/form-label/form-label.tsx:44]: Do you mean replace the old one and remove the Community's FormLabel?

Comment (kasparsuvi1) [reply libs/react-components/tedi/src/components/form/form-label/form-label.tsx:44]: Yes, not sure why we need duplicated components.

Comment (airikej) [reply libs/react-components/tedi/src/components/form/form-label/form-label.tsx:44]: Yes agreed, just forgot to do that, will make a task and replace them and make sure nothing will be broken

**Pull request merged**